Tests/IntegrationTests/Components Tests/PayTo/PayToComponentTests.swift

```diff
@@ -173,13 +172,33 @@ class PayToComponentTests: XCTestCase {
         XCTAssertNotNil(abnInputItem, "abn input field should exist")
     }
 
-    func test_organizationID_textfield_exists() throws {
+    func test_organizationID_textfield() throws {
+        // invalid
+        XCTAssertFalse(sut.organizationIdInputItem.isValid())
+        sut.organizationIdInputItem.value = " Hello"
+        XCTAssertFalse(sut.organizationIdInputItem.isValid())
+        sut.organizationIdInputItem.value = "Hello world "
+        XCTAssertFalse(sut.organizationIdInputItem.isValid())
+        sut.organizationIdInputItem.value = "A valid sentence?"
+        XCTAssertFalse(sut.organizationIdInputItem.isValid())
+        sut.organizationIdInputItem.value = "A!V"
+        XCTAssertFalse(sut.organizationIdInputItem.isValid())
+        
+        // valid
+        sut.organizationIdInputItem.value = "123123"
+        XCTAssertTrue(sut.organizationIdInputItem.isValid())
+        sut.organizationIdInputItem.value = "!@"
```

Comment:
This and following values are confusing to say the least!

Reply:
updated, they were auto generated values